Make -j(>=6) always fail as some job depends on config-host.h. Added following
patch in upstream to resolve it.

Could remove it in next upgrade if necessary.

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

commit f0acb38015409024683911380daa94cc974e4e0e
Author: Paul Brook <paul@codesourcery.com>
Date:   Fri Nov 26 18:46:03 2010 +0000

    Add missing dependency.
    
    Teach Makefile that cmd.o depends on a generated header (specifically
    config-host.h).
    
    Signed-off-by: Paul Brook <paul@codesourcery.com>
    (cherry picked from commit 6e14404aab26f74a448747d1e793ac16bde8a92b)
    
    Signed-off-by: Aurelien Jarno <aurelien@aurel32.net>

Index: qemu-0.13.0/Makefile
===================================================================
--- qemu-0.13.0.orig/Makefile	2011-01-18 09:45:45.000000000 +0800
+++ qemu-0.13.0/Makefile	2011-01-18 09:45:45.000000000 +0800
@@ -107,7 +107,7 @@
 ######################################################################
 
 qemu-img.o: qemu-img-cmds.h
-qemu-img.o qemu-tool.o qemu-nbd.o qemu-io.o: $(GENERATED_HEADERS)
+qemu-img.o qemu-tool.o qemu-nbd.o qemu-io.o cmd.o: $(GENERATED_HEADERS)
 
 qemu-img$(EXESUF): qemu-img.o qemu-tool.o qemu-error.o $(block-obj-y) $(qobject-obj-y)
 
